### ISSUE TITLE
why not syncbn in TinyViT ?

### ISSUE THREAD

**Issue by CYL0089** [microsoft/Cream#198]
In the paper:
![image](https://github.com/microsoft/Cream/assets/18739352/cdda691f-91ca-46d4-a2eb-0f1b73111a18)

but in the code:
![image](https://github.com/microsoft/Cream/assets/18739352/c02c214b-587c-4822-8855-1486348f4e9e)

and the pretrain and finetune command (in the [TinyViT/docs/TRAINING.md](https://github.com/microsoft/Cream/blob/b799630a29995163f282b15e2f38701160272fd1/TinyViT/docs/TRAINING.md)) not add the '--use-sync-bn'  

`python -m torch.distributed.launch --master_addr=$MASTER_ADDR --nproc_per_node 8 --nnodes=4 --node_rank=$NODE_RANK main.py --cfg configs/22k_distill/tiny_vit_21m_22k_distill.yaml --data-path ./ImageNet-22k --batch-size 128 --output ./output --opts DISTILL.TEACHER_LOGITS_PATH ./teacher_logits/

python -m torch.distributed.launch --nproc_per_node 8 main.py --cfg configs/22kto1k/tiny_vit_21m_22kto1k.yaml --data-path ./ImageNet --batch-size 128 --pretrained ./checkpoints/tiny_vit_21m_22k_distill.pth --output ./output`

So the batch-size is 128 actually?

**Comment by CYL0089** [comment]
Maybe I  miss something?

**Comment by wkcn** [comment]
Hi @CYL0089 , thanks for your attention  to our work!
The batch size per GPU is 128, and the number of GPUs is 32.

We did not try to enable SyncBN, but I think the batch size per GPU of 128 is enough for BatchNorm.

**Comment by CYL0089** [comment]
> Hi @CYL0089 , thanks for your attention to our work! The batch size per GPU is 128, and the number of GPUs is 32.
> 
> We did not try to enable SyncBN, but I think the batch size per GPU of 128 is enough for BatchNorm.

I see, thank you
